### ISSUE TITLE
Model is not updated for clickable event it only supports slide event

### ISSUE THREAD

**Issue by hemantpatil25** [seiyria/angular-bootstrap-slider#71]
Hi,

Model is not updated for clickable event it only supports slide event.

When i am clicking on slider the event is not triggered.

Please help me for this.

Regards,
Hemant

**Comment by hemantpatil25** [comment]
Any Updates on this?

**Comment by seiyria** [comment]
No, sorry, I don't actively develop this. If you figure something out, please feel free to send a pull request.

**Comment by evillemez** [comment]
This is killing me too... I'm looking for a solution, but no news yet: would love to know if anyone else out there has worked around it.

**Comment by evillemez** [comment]
I'm thinking that maybe this was just due to a version mismatch...  I had an override being loaded into my project, and once I ensured proper versions of this and bootstrap-slider were in, it seemed to start working as expected on click events.... so I dunno.  Couldn't begin to say when the mismatch started causing a problem, though, and I'm not 100% sure that's the answer, but it could be it.

**Comment by hemantpatil25** [comment]
I think it support click able event need to modify existing library.
Tomorrow I will try and let you know.

**Comment by fon60** [comment]
Please check if this PR will fix our issue. https://github.com/seiyria/angular-bootstrap-slider/pull/114

**Comment by alxgiraud** [comment]
+1

I had to add a `ng-click` directive with the same function as a workaround.

`<slider on-slide="update()" ng-click="update()">`
